ngrinder-groovy/src/main/java/org/ngrinder/http/RealHTTPRequest.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (c) 2012-present NAVER Corp.
+ *
+ * This file is part of The nGrinder software distribution. Refer to
+ * the file LICENSE which is part of The nGrinder distribution for
+ * licensing details. The nGrinder distribution is available on the
+ * Internet at https://naver.github.io/ngrinder
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ngrinder.http;
+
+import net.grinder.common.GrinderException;
+import net.grinder.plugin.http.HTTPPlugin;
+import net.grinder.plugin.http.HTTPPluginThreadState;
+import net.grinder.plugininterface.PluginThreadContext;
+import net.grinder.script.Statistics;
+import net.grinder.statistics.StatisticsIndexMap;
+import okhttp3.*;
+import okio.BufferedSource;
+import org.conscrypt.Conscrypt;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.net.ssl.HostnameVerifier;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.security.Security;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+public class RealHTTPRequest {
```

Comment:
What about InternalHttpRequest?

Reply:
We have to hide this class internally to avoid byte code instrument. So, I think that `InternalHttpRequest` makes sense. I'll change it.